app/models/application.rb

```diff
@@ -0,0 +1,7 @@
+class Application < ApplicationRecord
```

Comment:
I think we should probably rename this to `ScholarshipApplication` or something like that. `Application` might be confused with the actual rails application.